tests/acceptance/bashunit_execution_error_test.sh

```diff
@@ -35,25 +21,24 @@ function test_bashunit_when_a_execution_error() {
     printf " %s%s%s%s" "${color_red}" "$1" "${color_default}" "$2"
   }
 
-  fixture_start=$(
-    printf "Running ./tests/acceptance/fixtures/test_bashunit_when_a_execution_error.sh"
-    format_fail_title "✗ Failed" ": Error"
-    format_expect_title "Expected"
-    format_expect_value "'127'"
-    format_expect_title "to be exactly"
-    format_expect_value "'1'"
+  local fixture_start=$(
+    printf "%sRunning ./tests/acceptance/fixtures/test_bashunit_when_a_execution_error.sh%s\n" \
+      "${color_bold}" "${color_default}"
+    printf "%s✗ Error%s: Error\n" "${color_red}" "${color_default}"
+    printf "    %sline 4: invalid_function_name: command not found%s\n" "${color_dim}" "${color_default}"
   )
-  fixture_end=$(
+  local fixture_end=$(
     format_summary_title "Tests:     "
     format_summary_value "1 failed" ", 1 total"
     format_summary_title "Assertions:"
-    format_summary_value "1 failed" ", 1 total"
+    format_summary_value "0 failed" ", 0 total"
```

Comment:
This is intentional, because a `command not found`. Context:
```bash
function test_error() {
  # before we had this as a way to find out that a command did not exist
  invalid_function_name
  assert_general_error

  # now bashunit is clever enough to know that a command does not exist by itself, 
  # without the need of an assertion afterwards
  invalid_function_name arg1 arg2
}
```

For this reason, there are no assertions in this test. Just a failing test.